examples/fuzzing/linux_x8664/fuzz.c

```diff
@@ -10,13 +10,13 @@ int fun(int i)
     char *buf = malloc(SIZE);
     char buf2[SIZE];
 
-    while (*buf = getc(stdin) == 'A')
+    while ((*buf = getc(stdin)) == 'A')
     {
         buf[i++] = *buf;
     }
 
     strncpy(buf2, buf, i);
-    printf(buf2);
+    printf("%s", buf2);
```

Comment:
This is actually fixing an _Uncontrolled Format String_ vulnerability that could be triggered if `buf` was not limited only to `'A'` chars. Anyway, consider changing that to `puts(buf2)` that is functionally identical but much more efficient where there is no format string involved.

Reply:
Thanks! This has been changed.